code/processes/wdb.q

```diff
@@ -240,21 +241,32 @@ setcompression:{[compression] if[3=count compression;
 				]}
 resetcompression:{setcompression 16 0 0 }
 
+movetohdb:{[dw;hw;pt]
+  $[not(`$string[pt])in`$.os.list .os.pth -10 _ hw;
+      .[.os.ren;(dw;hw);{.lg.e[`mvtohdb;"Failed to move data from wdb ",x," to hdb directory ",y," : ",z]}[dw;hw]];
+      0=count .os.list .os.pth hw; 
+      [.[.os.ren;(dw,"/*";hw,"/.");{.lg.e[`mvtohdb;"Failed to move data from wdb ",x," to hdb directory ",y," : ",z]}[dw;hw]];.os.deldir dw]; 
```

Comment:
did you test this on windows? 

I have a feeling it might be better to do the move subdirectory at a time, and print a log comment around it